fineract-provider/src/main/java/org/apache/fineract/portfolio/charge/api/ChargesApiResourceSwagger.java

```diff
@@ -88,7 +88,7 @@ static final class GetChargesPaymentModeResponse {
             private GetChargesPaymentModeResponse() {}
 
             @Schema(example = "1")
-            public Long id;
+            public Integer id;
```

Comment:
This looks wrong to me...It should be Long based on the EnumOptionData line:36

Reply:
Done.